chapter_linear-networks/linear-regression-scratch_vn.md

```diff
@@ -296,7 +299,7 @@ def squared_loss(y_hat, y):
 ## Defining the Optimization Algorithm
 -->
 
-## *dịch tiêu đề phía trên*
+## Xác định Thuật toán tối ưu
```

Comment:
```suggestion
## Định nghĩa thuật toán tối ưu
```

Nhiều bạn dịch `Definine` là xác định lắm luôn. Này dịch là `định nghĩa` nhe!

Reply:
Như mình đã nói ở PR trước, vì mọi người đã dịch Defining là Xác định, thế thống nhất mình giữ nguyên cách dịch này. Nếu bạn muốn thay đổi có thể mở một PR riêng để mọi người bàn luận

Reply:
Hmm, anh không nhớ là mọi người thống nhất như vậy ở đâu. Em dẫn lại link.

`determine` (verb) hay `deterministic` (adj) mới là `xác định`. Ở đây đúng là `Định nghĩa` mà.

Reply:
> Như mình đã nói ở PR trước, vì mọi người đã dịch `Defining` là `Xác định` từ đầu nên thống nhất giữ nguyên cách dịch này. Nếu bạn muốn thay đổi có thể mở một PR riêng để mọi người bàn luận

Anh nghĩ mình phải quay lại sửa luôn. Anh hoàn toàn đồng ý với việc dịch là "định nghĩa". Cái này là định nghĩa hàm khi lập trình, không phải định nghĩa toán học.